ext/RastersStatsBaseExt/sample.jl

```diff
@@ -0,0 +1,92 @@
+Rasters.sample(x::RA.RasterStackOrArray, n::Integer; kw...) = Rasters.sample(Random.GLOBAL_RNG, x, n; kw...)
+@inline function Rasters.sample(
```

Comment:
I think we talked about it and I even wrote it in the docstring. We call StatsBase.sample under the hood, but we add a whole lot of keywords and totally change the output, so I think it would be confusing to add it as a method of StatsBase.sample. Plus `StatsBase.sample(x::Raster)` already works and could also still be useful after adding this.